tools/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+case $1 in
+-k) keep="keep"; shift;;
+*)  keep="";;
+esac
+set -o pipefail
+# yes | crew install buildessential $keep
+arch=`uname -m`
+for i in "$@"; do
+    version=`grep '\<version ['"'"'"]' /usr/local/lib/crew/packages/$i.rb | head -1 | sed -e 's/^.*version *['"'"'"]//' -e 's/['"'"'"].*$//'`
```

Comment:
Should we use `CREW_LIB_PREFIX` constant?

Reply:
This is shell script.  How can I use `CREW_LIB_PREFIX`?

Reply:
You can define a variable in shell with the same logic as ruby.  Different syntax but the logic to determine the directory is the same.

Reply:
But in this case, since all the crew stuff goes into `/usr/local/lib/crew` anyway, it's pointless.